racingcar-java/src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,82 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+class CalculatorTest {
+
+    private Calculator calculator = new Calculator();
+
+    @Test
+    @DisplayName("문자열계산기 덧셈")
+    void calculatorPlusTest() {
+        assertThat(calculator.calculator("2 + 3")).isEqualTo(5);
+    }
+
+    @Test
+    @DisplayName("문자열계산기 뺄셈")
+    void calculatorMinusTest() {
+        assertThat(calculator.calculator("3 - 1")).isEqualTo(2);
+    }
+
+    @Test
+    @DisplayName("문자열계산기 곱셈")
+    void calculatorMultiTest() {
+        assertThat(calculator.calculator("3 * 2")).isEqualTo(6);
+    }
+
+    @Test
+    @DisplayName("문자열계산기 나눗셈")
+    void calculatorDivideTest() {
+        assertThat(calculator.calculator("9 / 3")).isEqualTo(3);
+    }
+
+    @Test
+    @DisplayName("문자열계산기 null 테스트")
+    void calculatorNullTest() {
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() -> calculator.calculator(null)
+                ).withMessageMatching("입력값이 null 임");
```

Comment:
```suggestion
                .isThrownBy(() -> calculator.calculator(null))
                .withMessageMatching("입력값이 null 임");
```
)까지가 isThrownBy 라인이기때문에 한칸 올리는게 맞는 방법입니다 :)